src/screens/SignTransactionSheet.tsx

```diff
@@ -1399,12 +1399,11 @@ const DetailsCard = ({
 
   const showFunctionRow = meta?.to?.function || (methodName && methodName.substring(0, 2) !== '0x');
   const isContract = showFunctionRow || meta?.to?.created || meta?.to?.sourceCodeStatus;
-
+  const showTransferToRow = meta?.transferTo;
```

Comment:
since we only care about address here we should change this check to check for the address, also id prefer this to be a boolean instead of a value so lets would love a !!